python/surf/axi/_AxiLiteRingBuffer.py

```diff
@@ -28,59 +28,59 @@ def __init__(self, datawidth=32, **kwargs):
         ##############################
 
         self.add(pr.RemoteVariable(
-            name         = 'blen',
+            name         = 'BufferLength',
```

Comment:
@bengineerd Isn't this change (and the other remote variable name changes) going to break existing software?